gridfs/src/main/scala/GridFSTransport.scala

```diff
@@ -0,0 +1,49 @@
+// /*
+//  * Copyright (C) 2018-2019 Zengularity SA (FaberNovel Technologies) <https://www.zengularity.com>
+//  */
+package com.zengularity.benji.gridfs
+
+import reactivemongo.api.MongoConnection
+
+import reactivemongo.api.gridfs.GridFS
+import reactivemongo.api.BSONSerializationPack
+import scala.concurrent.{ ExecutionContext, Future }
+import scala.util.{ Failure, Success, Try }
+
+import com.zengularity.benji.exception.BenjiUnknownError
+import scala.util.Success
+
+final class GridFSTransport(driver: reactivemongo.api.MongoDriver, connection: MongoConnection, mongoUri: MongoConnection.ParsedURI) {
+  def close(): Unit = {
+    driver.close()
+  }
+
+  def gridfs(prefix: String)(implicit ec: ExecutionContext): Future[GridFS[BSONSerializationPack.type]] = {
+    mongoUri.db match {
+      case Some(name) =>
+        for {
```

Comment:
`for` est utile avec plus d'un `<-` (map), sinon un `.map` (plus explicite et concis dans ce cas)